Makefile

```diff
@@ -2,13 +2,15 @@ UNAME:=$(shell uname)
 
 # Install settings
 PREFIX= /opt/cartesi
-BIN_INSTALL_PATH= $(PREFIX)/bin
-LIB_INSTALL_PATH= $(PREFIX)/lib
-SHARE_INSTALL_PATH= $(PREFIX)/share
-IMAGES_INSTALL_PATH= $(SHARE_INSTALL_PATH)/images
-LUA_INSTALL_CPATH= $(PREFIX)/lib/lua/5.4
-LUA_INSTALL_PATH= $(PREFIX)/share/lua/5.4
-INC_INSTALL_PATH= $(PREFIX)/include/machine-emulator
+MACHINE_EMULATOR_VERSION:= $(shell make -sC src get-version)
+DEB_FILENAME= machine-emulator-v$(MACHINE_EMULATOR_VERSION).deb
```

Comment:
Should the deb name be `cartesi-machine-*` by default, like the package name?

Reply:
I like cartesi-machine better as well.
But I changed this from the last review to use the same naming the `.tar.gz`.

Reply:
changed to cartesi-machine.